### ISSUE TITLE
Ocurrence objects can't be adapted to ContentListingObject

### ISSUE THREAD

**Issue by erral** [plone/plone.app.event#410]
See https://github.com/plone/plone.restapi/issues/1809

**Comment by stevepiercy** [comment]
@erral this issue is open, but it looks like you fixed it in the linked issue and subsequent pull request. Can you verify? I'm prepping the big plone.app.event PLIP for the Alpine Sprint. Thank you!

**Comment by davisagli** [comment]
Yes, this was fixed in https://github.com/plone/plone.app.event/pull/411
